### ISSUE TITLE
Quick reference for Python '\w' incorrect

### ISSUE THREAD

**Issue by jnwatson** [firasdib/Regex101#1169]
## Bug Description

The current description for \w is:
`Matches any letter, digit or underscore. Equivalent to [a-zA-Z0-9_].`

For Python 3, unless the ASCII flag is set, matches Unicode word characters; this includes most characters that can be part of a word in any language, as well as numbers and the underscore. If the ASCII flag is set, matches any letter, digit or underscore. Equivalent to [a-zA-Z0-9_].

Python 2 has the same behavior, except it defaults to ASCII mode. It will match the default Python 3 behavior if the UNICODE flag is set.

The same changes apply to the quick reference for '\W'

**Comment by firasdib** [comment]
I assume this applies to all shorthands?

This does touch on a serious issue though. The site was initially implemented to support python 2.7, but now the majority has switched over to 3.x, which means it is due for an update. There are likely a lot of things that do not conform with python 3.x

**Comment by jnwatson** [comment]
The list of changed shorthands are '\w, \W, \b, \B, \d, \D, \s and \S' (from https://docs.python.org/3/library/re.html#re.ASCII)

I did a little research, and the only difference between Python 3.x and Python 2.7 I could find documented is dealing with locales and the default state of the UNICODE vs. ASCII flags.

Experimenting with the website, it looks like the "Python" flavor defaults to Python 3 behavior, in that random Cyrillic characters match `\w`.

In short, I don't think there's a lot of work to do. You might add a "Python 2" flavor selection, but that implies to me that there's a significant difference between the Python versions that just isn't there.

**Comment by firasdib** [comment]
I believe this is resolved with the new version. Please ping if still an issue.
